onboarding.md

```diff
@@ -0,0 +1,591 @@
+
+```properties
+web.http.port=9191
+```
+
+An example file can be found [here](samples/03-configuration/config.properties). Clean, rebuild and run the connector
+again, but this time passing the path to the config file:
+
+```properties
+java -Ddataspaceconnector.fs.config=/etc/eclipse/dataspaceconnector/config.properties -jar samples/03-configuration/build/libs/filsystem-config-connector.jar
+```
+
+Observing the log output we now see that the connector's REST API is exposed on port `9191` instead:
+
+```bash
+INFO 2021-09-07T08:43:04.476254 Registered Web API context at: /api/*
+INFO 2021-09-07T08:43:04.503543 HTTP listening on 9191                  <--this is the relevant line
+INFO 2021-09-07T08:43:04.750674 Started Web extension
+```
+
+### Add your own configuration value
+
+Lets say we want to have a configurable log prefix in our health REST endpoint. The way to do this involves two steps:
+
+1. add the config value to the `config.properties` file
+2. access and read the config value from code
+
+#### 1. Add the config value
+
+Simply add a new line with an arbitrary key to your `config.properties`:
+
+```properties
+edc.samples.03.logprefix=MyLogPrefix
+```
+
+#### 2. Access the config value
+
+The `ServiceExtensionContext` exposes a method `getSettings(String, String)` to read settings (i.e. config values)'.
+Modify the code from the `HealthEndpointExtension.java` as shown below (use the one from the `samples/03-configuration`
+of course):
+
+```java
+public class HealthEndpointExtension implements ServiceExtension {
+    private static final String LOG_PREFIX_SETTING = "edc.samples.03.logprefix"; // this constant is new
+
+    @Override
+    public Set<String> requires() {
+        return Set.of("edc:webservice");
+    }
+
+    @Override
+    public void initialize(ServiceExtensionContext context) {
+        var logPrefix = context.getSetting(LOG_PREFIX_SETTING, "health"); //this line is new
+        var webService = context.getService(WebService.class);
+        webService.registerController(new HealthApiController(context.getMonitor(), logPrefix));
+    }
+}
+```
+
+Next, we must modify the constructor signature of the `HealthApiController` class and store the `logPrefix` as variable:
+
+```java
+
+@Consumes({MediaType.APPLICATION_JSON})
+@Produces({MediaType.APPLICATION_JSON})
+@Path("/")
+public class HealthApiController {
+
+    private final Monitor monitor;
+    private final String logPrefix;
+
+    public HealthApiController(Monitor monitor, String logPrefix) {
+        this.monitor = monitor;
+        this.logPrefix = logPrefix;
+    }
+
+    @GET
+    @Path("health")
+    public String checkHealth() {
+        monitor.info(String.format("%s :: Received a health request", logPrefix));
+        return "I'm alive!";
+    }
+}
+```
+
+There are a few things worth mentioning here:
+
+- things like configuration value names should be implemented as constants, e.g. `LOG_PREFIX_SETTING` and should have a
```

Comment:
* why constants? (guess most will do it that way, but is there a hard requirement?)
* if consisten naming is required we should provide a naming scheme